fingerprint/infra/scanner/src/main/java/com/simprints/fingerprint/infra/scanner/v2/tools/WrapErrorFromScanner.kt

```diff
@@ -31,3 +33,12 @@ fun wrapErrorFromScanner(e: Throwable): Throwable = when (e) {
         e
     }
 }
+
+suspend fun <T> executeSafely(block: suspend () -> T): T {
```

Comment:
The name is a bit confusing, since the method still throws the error, just wrapped and logged. What makes it safer?